dev/WebView2/InteractionTests/WebView2Tests.cs

```diff
@@ -2179,6 +2077,78 @@ public void NonAsciiUriTest()
             }
         }
 
+        [TestMethod]
+        [TestProperty("TestSuite", "D")]
+        public void CreateWebView2OffTreeTest()
```

Comment:
> public void CreateWebView2OffTreeTest() [](http://example.com/codeflow?start=7&length=40)

This basic scenario is covered in BasicCoreObjectCreationAndDestructionTest (specifically Preload Test 1 & 2 subtests). But here you also valiate input, which I agree is very valuable. Maybe change the name to emphasize that (eg "OffTreeWebViewInputTest")? If that's too much trouble, just adding a comment about this here is good too.